gobblin-iceberg/src/test/java/org/apache/gobblin/iceberg/writer/HiveMetadataWriterTest.java

```diff
@@ -256,6 +258,54 @@ public void testHiveWriteRewriteFileGMCE() throws IOException {
     });
   }
 
+  @Test(dependsOnMethods = {"testHiveWriteRewriteFileGMCE"}, groups={"hiveMetadataWriterTest"})
+  public void testHiveWriteDeleteFileGMCE() throws IOException, TException {
+    gmce.setTopicPartitionOffsetsRange(null);
+    Map<String, String> registrationState = gmce.getRegistrationProperties();
+    registrationState.put("additional.hive.database.names", dedupedDbName);
+    registrationState.put(HiveMetaStoreBasedRegister.SCHEMA_SOURCE_DB, dbName);
+    gmce.setRegistrationProperties(registrationState);
+    gmce.setSchemaSource(SchemaSource.NONE);
+    gmce.setOldFilePrefixes(Lists.newArrayList(dailyDataFile.toString()));
+    gmce.setOperationType(OperationType.drop_files);
+    gobblinMCEWriter.writeEnvelope(new RecordEnvelope<>(gmce,
```

Comment:
Could you add a couple more comments in the tests just to make it easy to understand what is being tested, here are we testing dropping a partition that doesn't exist?